hardhat.config.ts

```diff
@@ -12,6 +12,15 @@ import {
     defaultLocalhostNetworkParams,
 } from 'hardhat/internal/core/config/default-config';
 
+const managerAccount =
+    Number(process.env.IEXEC_VOUCHER_MANAGER_ACCOUNT_INDEX) ||
+    process.env.IEXEC_VOUCHER_MANAGER_ACCOUNT_ADDRESS ||
```

Comment:
Not right now, but we will for sure at deployment.

Reply:
I'm not sure we will use localhost for that

Reply:
Correct yes. But it will be the same env variable `IEXEC_VOUCHER_MANAGER_ACCOUNT_ADDRESS`.
I can add `bellecour: managerAccount` to the config. I didn't because I think we can do it separately when we will start testing deployment.

Reply:
Replied here ! https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/34#discussion_r1642845088